tests/fdb/test_fdb_mac_expire.py

```diff
@@ -0,0 +1,251 @@
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        duthost.shell(argv=["sonic-clear", "fdb", "all"])
+
+        yield
+
+        duthost.shell(argv=["sonic-clear", "fdb", "all"])
+
+    @pytest.fixture(scope="class", autouse=True)
+    def validateDummyMacAbsent(self, duthost):
+        """
+            Validates that test/dummy MAC entry is absent before the test runs 
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        assert self.__getFdbTableCount(duthost, self.DUMMY_MAC_PREFIX) == 0, "Test dummy MAC is already present"
+
+    @pytest.fixture(scope="class", autouse=True)
+    def prepareDut(self, request, duthost):
+        """
+            Prepare DUT for FDB test
+
+            It update the fdb_aging_time value, update the swss configuration, and restore SWSS configuration afer
+            test completes
+
+            Args:
+                request (Fixture): pytest request object
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        fdbAgingTime = request.config.getoption('--fdb_aging_time')
+
+        self.__deleteTmpSwitchConfig(duthost)
+        duthost.shell(argv=["docker", "cp", "swss:/etc/swss/config.d/switch.json", "/tmp"])
+        duthost.replace(
+            dest='/tmp/switch.json',
+            regexp='"fdb_aging_time": ".*"',
+            replace='"fdb_aging_time": "{0}"'.format(fdbAgingTime),
+            backup=True
+        )
+        duthost.shell(argv=["docker", "cp", "/tmp/switch.json", "swss:/etc/swss/config.d/switch.json"])
+        self.__loadSwssConfig(duthost)
+
+        yield
+
+        result = duthost.find(path=["/tmp"], patterns=["switch.json.*"])
+        if result["matched"] > 0:
+            duthost.shell(argv=["docker", "cp", result["files"][0]["path"], "swss:/etc/swss/config.d/switch.json"])
+            self.__loadSwssConfig(duthost)
+        self.__deleteTmpSwitchConfig(duthost)
+
+    def testFdbMacExpire(self, request, testbed, duthost, ptfhost):
+        """
+            TestFdbMacExpire Verifies FDb aging timer is respected
+
+            The test updates fdb_aging_time value, restarts swssconfig in order to pickup the new value,
+            populated the FDB table with dummy MAC entry, and then waits for fdb_aging_time and makes sure
+            FDB entry with dummy MAC is cleared.
+
+            Args:
+                request (Fixture): pytest request object
+                testbed (Fixture, dict): Map containing testbed information 
+                duthost (AnsibleHost): Device Under Test (DUT)
+                ptfhost (AnsibleHost): Packet Test Framework (PTF)
+
+            Returns:
+                None
+        """
+        topo = testbed["topo"]["name"]
+        if topo not in self.SUPPORTED_DUT_TOPOS:
```

Comment:
You could check testbed['topo']['type'] to be 't0'. Should all T0 topology support this test?

Reply:
Will change it. Re: supported topo'ss, these are the supported T0 topo's:
```
    SUPPORTED_DUT_TOPOS = ["t0", "t0-64", "t0-64-32", "t0-116", "t0-52"]
```
These is subset of the T0 topo's. Will check for T0 only as there nothing should prevent this from running on all T0 variants.